app/Filament/Resources/UserResource/Pages/EditUser.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\UserResource\Pages;
+
+use App\Filament\Resources\UserResource;
+use App\Site\Models\User;
+use Filament\Resources\Pages\EditRecord;
+
+class EditUser extends EditRecord
+{
+    protected static string $resource = UserResource::class;
+
+    public static function getNavigationLabel(): string
+    {
+        return __('Edit');
+    }
+
+    public function getSubheading(): ?string
+    {
+        /** @var User $record */
+        $record = $this->getRecord();
+
+        return '[' . $record->ID . '] ' . $record->User;
+    }
+
+    protected function getHeaderActions(): array
+    {
+        return [
+        ];
+    }
```

Comment:
Changes to the Legacy permissions level are not captured in the audit log, and do not trigger the related functionality (i.e. changing the value to Banned does not clear out the authentication fields [Password, AppToken, etc]).

Reply:
Removed input, see https://github.com/RetroAchievements/RAWeb/pull/2165#discussion_r1465694276

Banning users will be a new action in the admin panels which does not rely on `Permissions` anymore but on a `banned_at` timestamp which will behave similarly to the soft-delete `deleted_at` timestamp (global eloquent model scope will be introduced for that).